Client/Frontend/Strings.swift

```diff
@@ -2056,6 +2056,36 @@ extension String {
         value: "Cannot Open Page",
         comment: "Title of the message shown when the user attempts to navigate to an invalid link.",
         lastUpdated: .unknown)
+    public static let CouldntDownloadWallpaperErrorTitle = MZLocalizedString(
+        "Wallpaper.Download.Error.Title.v106",
+        value: "Couldn't Download Wallpaper",
```

Comment:
This should use a proper apostrophe (`’`). For example, `Wallpaper.Change.Error.Title.v106` already does.